src/corner_kick/README.md

```diff
@@ -0,0 +1,55 @@
+# Corner Kick
+
+Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+
+## How to use
+
+You can start the visualizer by running:
+
+```
+roslaunch corner_kick corner_kick.launch
+```
+
+## Contribute
+
+### Running the dev server
+
+You can test your code by running:
+
+```
+roslaunch rosbridge_server rosbridge_websocket.launch
+```
+
+and in another window:
+
+```
+yarn
+yarn start
+```
+
+A window should open with the application in development mode.
+
+## Project Structure
+
+The project is defined as follow:
+
+-   config - contains webpack (build tool) configuration files
+-   launch - contains roslaunch file
+-   src
+    -   main - contains code for the main process
+    -   renderer - contains code for the rendering process
+        -   components
+            -   containers - contains React stateful components
+            -   ui - contains repeatable UI components
+        -   modules - contains high-level UI components (logger, visualizer, etc.)
+    -   shared - contains code shared by the two process
+        -   ros - contains the service communicating with ROS
+            -   types - contains ROS message type schema
+-   .prettierrc - configuration for autoformating tool
+-   CMakeLists.txt - cmake build script
+-   module.d.ts - definition file for non-typescript libraries
+-   package.json - specifies dependencies, dev and build scripts.
+-   package.xml - specifies ros dependencies
+-   tsconfig.json - configuration file for typescript compiler
+-   tslint.json - configuration for typescript linter
+-   yarn.lock - autogenerated lock file for javascript dependencies
```

Comment:
Just bumping the below comment from the last round as it looks like it might have been missed:

> Sorry, I meant use the " ` " character around them, ie:
> ``` suggestion
> - `yarn.lock` - autogenerated lock file for javascript dependencies
> ```